hasura/metadata/cron_triggers.yaml

```diff
@@ -6,3 +6,16 @@
   headers:
   - name: verification_key
     value_from_env: HASURA_EVENT_SECRET
+- name: dailyUpdate
+  webhook: '{{HASURA_API_BASE_URL}}/cron/dailyUpdate'
+  schedule: 0 21 * * *
+  include_in_metadata: true
+  payload: {}
+  retry_conf:
+    num_retries: 0
```

Comment:
why 0 here?

Reply:
I'm trying to err towards simplicity. If we want to use retries here, we'd need to index the status of various updates to avoid resending them.